net/asterisk-15.x/Makefile

```diff
@@ -840,6 +849,7 @@ $(eval $(call BuildAsterisk15Module,codec-g726,Signed linear to G.726 translatio
 $(eval $(call BuildAsterisk15Module,codec-gsm,linear to GSM translation,translate between signed linear and GSM,,,codec_gsm,,))
 $(eval $(call BuildAsterisk15Module,codec-ilbc,linear to ILBC translation,translate between signed linear and ILBC,,,codec_ilbc,,))
 $(eval $(call BuildAsterisk15Module,codec-lpc10,Linear to LPC10 translation,translate between signed linear and LPC10,,,codec_lpc10,,))
+$(eval $(call BuildAsterisk15Module,codec-opus-open-source,OPUS codec,Translate between signed linear and Opus (Open Codec),+libopus,,codec_opus_open_source,,))
```

Comment:
Why not also add format_ogg_opus_open_source and format_vp8? Debian packages them as well, see https://packages.debian.org/stretch/amd64/asterisk-opus/filelist

Reply:
Hello Andre, makes sense, thanks for explaining.